packages/metal-isomorphic/test/isomorphic.js

```diff
@@ -0,0 +1,22 @@
+import Component from 'metal-component';
+import MyComponent from './fixtures/MyComponent';
+import MyJSXComponent from './fixtures/MyJSXComponent';
+import { assert } from 'chai';
```

Comment:
Wouldn't we rather use `jest` snapshots here? Since we're already using it in other places and we won't be testing this on browsers... that way we have the same test dep everywhere.

Reply:
Sure we can do that.

Reply:
Do you think we can make that improvement down the line? There are a lot of places in this repo that we could be using jest snapshots, it would be nice to do that migration all at once under a separate issue.

Reply:
Sure, we can do that